### PR TITLE
[rotel] New channel for other commands provided by the API

### PR DESCRIPTION
**Pull request by lolodomo**
Models covered by this PR:
A11, A12, A14, CD11, CD12, RA11, RA12, RA1570, RA1572, RC1570, RC1572, RCD1572, P5, X3, X5,
RSP-1066, RSP-1068, RSP-1069, RSP-1098, RSP-1570, RSP-1572,
RSX-1055, RSX-1056, RSX-1057, RSX-1058, RSX-1065, RSX-1067, RSX-1550, RSX-1560, RSX-1562

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java [hunk 1]
```diff
@@ -323,50 +544,50 @@ public enum RotelCommand {
...
     /**
-     * Get the command name
+     * Get the command label
      *
-     * @return the command name
+     * @return the command label
      */
-    public String getName() {
-        return name;
+    public String getLabel() {
```
Comment (jlaur) [inline bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java:589]: Consider overriding `toString()` also for simplified logging.

**Review by jlaur — APPROVED** [review]
LGTM, thanks! One minor suggestion to consider.

File: bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java:589]: I am taking a look...

Comment (lolodomo) [reply bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java:590]: Done

**Comment by lolodomo** [comment]
Thank you.

**Pull request merged**